manifests/install_integration.pp

```diff
@@ -0,0 +1,20 @@
+define datadog_agent::install_integration (
+  Enum['present', 'absent'] $ensure           = 'present',
+  String                    $integration_name = undef,
+  String                    $version          = undef,
+){
+
+  include datadog_agent
+
+  if $ensure == 'present' {
+    $command = 'install'
+  } else {
+    $command = 'remove'
+  }
+
+  exec { "${command} ${integration_name}==${version}":
```

Comment:
This has a slight problem, given that if a customer installs an integration, this will install it with every run, making the module chatty. Ideally we'd want to only add the resource if the integration has not been installed (same goes for `absent`). 

You might be able to check for the existence of `/opt/datadog-agent/embedded/lib/python<PYVERSION>/site-packages/datadog_<integration>-<version>.dist-info` to decide if you go forward with the execution or not. Or you could ask pip.

Reply:
The `integration` command is idempotent, and exec only prints if the command ends with a non-zero status (so it won't make extra work and it won't be chatty).

I find it way cleaner to rely on the `integration` command doing the check than us digging into some (version-dependent) path to check for some internal pip files.

Reply:
I wasn't sure if `integration` was idempotent, it's good to hear it is. I think this would still make the module chatty though, signaling resource changes when no real resource has changed or should've been changed. If you can verify it's not the case, then I naturally don't see a need to change anything - but if it does, we probably should reconsider the approach.